lib/api/api.js

```diff
@@ -42,8 +43,9 @@ auth.setHandler(vault);
 
 const api = {
     callApiMethod(apiMethod, request, log, callback, locationConstraint) {
-        // no need to check auth on website requests
-        if (apiMethod === 'websiteGet' || apiMethod === 'websiteHead') {
+        // no need to check auth on website or cors preflight requests
+        if (apiMethod === 'websiteGet' || apiMethod === 'websiteHead' ||
+        apiMethod === 'corsGetResponse') {
```

Comment:
thanks for bringing this up to me.. it does look like AWS ignores bad credentials:
https://github.com/scality/S3/pull/518/commits/7ec42654715611ac98a65de431185ff5dcfb9062